google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/executor/AppEngineStandardRunTask.java

```diff
@@ -62,7 +67,9 @@ public void execute(ProcessStartListener startListener) {
         .startListener(startListener)
         .build();
 
-    CloudSdkAppEngineDevServer devServer = new CloudSdkAppEngineDevServer(sdk);
+    AppEngineDevServer devServer = useDevappserver2
```

Comment:
just curious: if the common lib uses the default name for devAppServer2, and explicitly calles out devAppServer1 (e.g. CloudSdkAppEngineDevServer1), why did you do it the other way around? does it have something to do with the default value we set?